### PR TITLE
pg_isready: add page

### PR DESCRIPTION
**Pull request by thamaraiselvam**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
@thamaraiselvam Welcome to tldr, always select the base branch as `main` as all the workflows will run on this branch only. `master` branch is kept for backwards compatibility and will soon be fully removed. I will review this PR now.

File: pages/common/pg_isready.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# pg_isready
+
+> Check the connection status of a PostgreSQL server.
+> More information: <https://www.postgresql.org/docs/current/app-pg-isready.html>.
+
+- Check connection with hostname and port:
```
Comment (kbdharun) [inline pages/common/pg_isready.md:6]:
```suggestion
- Check connection with a specific hostname and port:
```

**Comment by thamaraiselvam** [comment]
Thanks for your suggestions @kbdharun. I have resolved them.

File: pages/common/pg_isready.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# pg_isready
+
+> Check the connection status of a PostgreSQL server.
+> More information: <https://www.postgresql.org/docs/current/app-pg-isready.html>.
+
+- Check connection with a specific hostname and port:
+
+`pg_isready -h {{hostname}} -p {{port}}`
```
Comment (navarroaxel) [inline pages/common/pg_isready.md:8]: We prefer long arguments if available
```suggestion
`pg_isready --host={{hostname}} --port={{port}}`
```

File: pages/common/pg_isready.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Run connection status quietly:
```
Comment (navarroaxel) [inline pages/common/pg_isready.md:10]: Perhaps this better
```suggestion
- Check connection status displaying a message only when the connection fails:
```

File: pages/common/pg_isready.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+`pg_isready --quiet`
```
Comment (navarroaxel) [inline pages/common/pg_isready.md:12]: Is `--host` required?

Comment (thamaraiselvam) [reply pages/common/pg_isready.md:12]: host is not mandatory. default host is `localhost`.

File: pages/common/pg_isready.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# pg_isready
+
+> Check the connection status of a PostgreSQL server.
+> More information: <https://www.postgresql.org/docs/current/app-pg-isready.html>.
+
+- Check connection with a specific hostname and port:
```
Comment (navarroaxel) [inline pages/common/pg_isready.md:6]: If an argument-less version is accepted then:
```suggestion
- Check connection:

`pg_isready`

- Check connection with a specific hostname and port:
```

File: pages/common/pg_isready.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+`pg_isready --host={{hostname}} --port={{port}}`
+
+- Check connection status displaying a message only when the connection fails:
```
Comment (navarroaxel) [inline pages/common/pg_isready.md:10]: To make all the example descriptions homogeneous
```suggestion
- Check connection displaying a message only when the connection fails:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**